### ISSUE TITLE
Fails to create directories

### ISSUE THREAD

**Issue by SkylarPlayz348** [Twingamerdudes/CreatorOS#5]
After I create a user this happens
![image](https://user-images.githubusercontent.com/73512621/233854275-5046b2af-577f-498d-b034-9fb24d9f22a7.png)

**Comment by Twingamerdudes** [comment]
Is this from the current version 0.0.1 or the source code build?

**Comment by SkylarPlayz348** [comment]
Version 0.0.1, and this happens on both virtualbox and vmware for me

**Comment by Twingamerdudes** [comment]
Yeah, it looks like 0.0.1 is bugged for now I would recommend compiling from source code (main branch) to fix any errors and get access to more features as well.

**Comment by SkylarPlayz348** [comment]
Ok how would I do that. I'm not familiar with C#. I'm trying to learn but haven't started yet.

**Comment by Twingamerdudes** [comment]
**Edit: After trying this out myself it doesn't work, it needs a more specific version of cosmos that I don't even know.**

You need a specific version of cosmos to compile this project, download it here: https://drive.google.com/drive/folders/1Dfkqp3Of5r_aKUwezP3d9qOs0DE9cNKW?usp=sharing (make sure you install visual studio with c# for desktop apps enabled). Then download the project. Open it in visual studio. Then hit the button at the top bar with the green arrow that says cosmos.

**Comment by SkylarPlayz348** [comment]
Do I just open the `.sln` file or do I open the folder and do I need a certain version of Visual Studio. I have 2022 and I'm pretty new to working with VS.
